frontend/src/features/permits/pages/Application/components/review/ReviewPermitDetails.tsx

```diff
@@ -102,6 +105,14 @@ export const ReviewPermitDetails = ({
             )}
           />
         </Box>
+
+        {isPermitStartOrExpiryDateInPast(startDate, expiryDate) && (
+          <Box className="permit-error-banner">
+            <ErrorBcGovBanner
+              msg={"Start Date and/or Permit Expiry Date is in the past."}
```

Comment:
There are existing validation messages inside validation_messages.json and validationMessages.ts (which basically is a wrapper for validation_messages.json), and inside validation_messages.json under "date", you can add the following:
```
"date": {
    "defaultMessage": ...,
    "start: {...},
    "expiry": {
        "beforeStart": {...},
        "past": {
            "defaultMessage": "Start Date and/or Permit Expiry Date is in the past."
         }
    }
},
```
and then add a method in validationMessage.ts that returns this message (ie. do something similar to the method "invalidPastStartDate", except name it "invalidPastStartAndExpiryDate" or something like that)